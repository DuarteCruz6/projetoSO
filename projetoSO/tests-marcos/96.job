BACKUP
WAIT 2000
WRITE [(m6,54)(v7,60)(w9,7)(f5,28)]
BACKUP
WRITE [(u2,19)(p0,90)(c7,46)]
SHOW
WRITE [(z6,96)(v1,70)(e7,36)]
READ [m6]
DELETE [w9,p0]
WRITE [(t8,47)]
WRITE [(s1,92)(e3,12)(y3,22)(o5,17)(j8,93)]
WRITE [(u5,39)(y5,23)(k0,9)(u3,6)(r2,0)]
READ [o5]
WRITE [(j2,60)(h8,73)]
DELETE [s1,y5,t8]
SHOW
WRITE [(z2,82)(e2,63)(u7,64)]
WAIT 2000
DELETE [e3]
WRITE [(i1,92)(m7,31)(t6,64)(i3,32)]
READ [u2]
WRITE [(n5,61)(v0,35)]
WRITE [(x2,43)]
WRITE [(m1,12)(l3,30)(x1,64)(k6,12)(p7,82)]
READ [u7]
WRITE [(x0,94)(g8,90)]
WRITE [(f2,1)]
WRITE [(u4,68)(l8,62)(w1,90)(a0,9)]
WAIT 2000
WRITE [(o4,32)(p6,44)(d6,78)]
WRITE [(z1,83)]
READ [v1]
WRITE [(x7,91)(q9,18)(s7,35)(z3,88)(f0,93)]
WRITE [(u9,91)(n8,96)]
WRITE [(w8,78)]
DELETE [u3,v7,e2,l8]
READ [g8]
WRITE [(k2,70)]
DELETE [m7]
WRITE [(h5,94)(b5,69)]
WRITE [(i7,87)(a7,92)]
SHOW
WRITE [(b2,75)(u3,9)(x9,69)(c8,98)(d9,10)]
DELETE [z2,j2,u7,q9]
WRITE [(q6,92)(z5,88)(s1,39)]
READ [b2]
WRITE [(h2,49)(q4,22)(n2,8)(m7,89)]
READ [i7]
WRITE [(l4,1)]
READ [z3]
WAIT 1000
WRITE [(u8,94)(a3,19)(m3,82)(x8,54)(c4,69)]
WRITE [(s5,6)(y4,76)]
WRITE [(p0,61)(i0,47)(y5,25)]
WRITE [(q0,48)]
