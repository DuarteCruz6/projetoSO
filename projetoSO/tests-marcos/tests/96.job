SHOW
WRITE [(y0,20)(h2,63)(x0,0)]
BACKUP
WRITE [(o3,98)(n4,32)(x3,57)(x5,87)]
DELETE [x5,x0]
READ [h2]
DELETE [x3,y0,h2]
READ [n4]
WRITE [(t9,73)(g0,41)]
DELETE [t9]
WAIT 1000
READ [o3]
READ [g0]
READ [o3]
WRITE [(c4,59)(f1,8)]
READ [g0]
READ [n4]
DELETE [c4,o3,n4,f1,g0]
WRITE [(o7,96)(k7,36)]
WRITE [(k5,70)(c1,65)(j2,28)(h8,30)(g5,60)]
WAIT 2000
WRITE [(c5,66)(b8,66)]
WRITE [(u4,67)(r5,84)(e5,74)(h2,46)(k2,2)]
DELETE [h2]
WRITE [(m3,17)(v1,100)(a1,87)]
READ [o7]
WRITE [(k8,93)]
WRITE [(h0,78)(x8,40)(y2,50)]
WRITE [(o3,94)(t8,95)]
WRITE [(q8,59)(x5,24)(p3,50)]
WRITE [(p6,92)(b1,90)(y3,2)]
WRITE [(h6,72)(n7,9)(e8,64)(m8,35)]
WRITE [(l8,13)(a3,53)(a0,12)]
DELETE [y3,e5,m8]
READ [x8]
DELETE [h8,n7,a3,k7,g5]
READ [o3]
WRITE [(w1,50)(i5,87)(l4,100)]
WRITE [(q3,87)(y6,89)(k7,38)(j9,0)(s1,34)]
WRITE [(e2,48)(r0,57)(s5,17)(n5,76)]
WRITE [(c0,96)]
READ [c5]
WRITE [(g1,70)(s0,21)(f0,88)]
SHOW
WRITE [(i1,9)(i4,100)]
DELETE [k5]
DELETE [g1,e2,s5]
WRITE [(v7,69)(d6,83)]
WRITE [(j1,51)(n3,88)(g8,69)(e7,91)(h1,65)]
WRITE [(u3,65)(m4,22)]
WRITE [(o1,7)(f2,57)(x7,57)(z0,68)]
WRITE [(z9,83)(x9,10)]
