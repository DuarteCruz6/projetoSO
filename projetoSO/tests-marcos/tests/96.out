[(h2,63)]
[(n4,32)]
[(o3,98)]
[(g0,41)]
[(o3,98)]
[(g0,41)]
[(n4,32)]
[(o7,96)]
[(x8,40)]
[(o3,94)]
[(c5,66)]
(a0, 12)
(a1, 87)
(b1, 90)
(b8, 66)
(c0, 96)
(c5, 66)
(c1, 65)
(e2, 48)
(e8, 64)
(f0, 88)
(g1, 70)
(h6, 72)
(h0, 78)
(i5, 87)
(j9, 0)
(j2, 28)
(k7, 38)
(k8, 93)
(k2, 2)
(k5, 70)
(l4, 100)
(l8, 13)
(m3, 17)
(n5, 76)
(o3, 94)
(o7, 96)
(p6, 92)
(p3, 50)
(q3, 87)
(q8, 59)
(r0, 57)
(r5, 84)
(s0, 21)
(s5, 17)
(s1, 34)
(t8, 95)
(u4, 67)
(v1, 100)
(w1, 50)
(x5, 24)
(x8, 40)
(y6, 89)
(y2, 50)
